src/welcome/index.js

```diff
@@ -17,8 +18,7 @@ const github = require('@actions/github');
       return;
     }
 
-    const footer = `<p>If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="https://discord.com/invite/jZQs6Wu">Discord</a> chat and our <a href="https://github.com/EddieJaoudeCommunity">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.</p>
-      `;
+    const footer = `<p>` + prFooter + `</p>`;
```

Comment:
I think this would be better formatting...

```suggestion
    const footer = `<p>footer</p>`; 
```

I will put another comment for the variable rename above

Reply:
Okay, and Sorry